### DISCUSSION TITLE
Adding extra value to the Accordion component

### DISCUSSION THREAD

**Discussion by lcordobapuerto** [dxc-technology/halstack-react#2118] [Ideas]
In recent projects where we've worked on designing complex interfaces for product teams, we found ourselves wishing for a bit more flexibility in the Accordion component, particularly in the header. While we agree that the content of the header should be clearly defined, we'd like to explore the possibility of adding some components that would undoubtedly bring significant value to the component itself. Here's an example of an Accordion with a badge in the header, which could indicate the method by which the API retrieves data for a specific product.
![imagen (10)](https://github.com/user-attachments/assets/eaa702da-ae79-48eb-994e-3e01940476d8)

Thanks!

**Comment by raquelarrojo** [comment]
I like it. I think it is a good way to provide additional information.
I took a look to other Design Systems and they provide a similar solution for this.

Braid:
![image](https://github.com/user-attachments/assets/7a322aa6-2b91-48da-8322-e206f182e805)

Lightning:
![image](https://github.com/user-attachments/assets/870a8eab-95ce-4caa-8faf-ddd8159d79f7)

**Comment by raquelarrojo** [comment]
This has already been implemented:
https://github.com/dxc-technology/halstack-react/pull/2157
